.circleci/config.yml

```diff
@@ -458,16 +458,15 @@ jobs:
                 exit 1
               fi
       - go/mod-download-cached
-      - run: make
-      - run: make package
-      - run: make clean
+      - run: make generate-client
       - run:
           name: Creating artifacts directory
           command: mkdir /tmp/artifacts
       - go/test:
           coverpkg: "./api/...,./client/...,./cmd/...,./pkg/..."
           verbose: true
           coverprofile: "/tmp/artifacts/cover.out"
+          parallel: "3"
```

Comment:
I think this will actually decrease parallelism.  The job is running on `Linux Large`, which has 4 CPUs [1].  `go test`, by default, runs in parallel as many tests as there are CPUs on the machine running it, so I'd expect it to be running four tests in parallel, today.

[1] https://circleci.com/docs/configuration-reference/#resourceclass

Reply:
The go/test orb is doing something behind the scenes I don't feel is very intuitive. If you look at the actual go command that it runs in main right now it is something like this: `go test  -p 1 ./...`, which effectively constrains it to one test at a time.

Three was chosen because the executor this is running on `go_cimg` doesn't specify a resource class - so as I understand it, it is defaulted to medium (2 vCPUs). The 1.5x ratio was kind of arbitrarily chosen based on experience and a hunch that a lot of our test time is comprised of sleeping anyhow.

Reply:
Gotcha.  My fault, I didn't see it was dealing with the `test` job, not with `main_tests_minikube_local_cluster`.